### PR TITLE
Remove GCE autoscaling metrics from GCM sink

### LINKED ISSUE #1487: Remove GCE autoscaling metrics from GCM sink
GCE Autoscaler is not support in Kubernetes for a while. Let's remove metrics exported to Stackdriver for its purposes.

### PR DESCRIPTION
**Pull request by bskiba**
Fix #1487. Remove GCE autoscaling metrics from GCM sink. GCE Autoscaler has not
been supported in Kubernetes for a while. Also remove the `metrics`
parameter of GCM sink configuration, which was previously used to enable
collection of only GCE autoscaling metrics.

### CODE REVIEW

**Comment by bskiba** [comment]
Bump to verify cla.

File: metrics/core/metrics.go [hunk 1]
```diff
@@ -73,17 +73,6 @@ var LabeledMetrics = []Metric{
 	MetricFilesystemInodesFree,
 }
 
-var NodeAutoscalingMetrics = []Metric{
-	MetricNodeCpuCapacity,
```
Comment (piosz) [inline metrics/core/metrics.go]: Let's leave metrics:
```
MetricNodeCpuCapacity,
MetricNodeMemoryCapacity,		
MetricNodeCpuAllocatable,		
MetricNodeMemoryAllocatable,
```
which are generally useful.

Let's also rename `NodeAutoscalingEnricher` to just `NodeEnricher`

**Comment by piosz** [comment]
@mwielgus

**Comment by piosz** [comment]
Closing for now as obsolete.

**Pull request closed without merging**